enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -142,6 +144,13 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
       MojoExecution execution = (MojoExecution) helper.evaluate("${mojoExecution}");
       RepositorySystemSession repositorySystemSession = session.getRepositorySession();
 
+      ImmutableList<String> repositoryUrls =
+          project.getRemoteProjectRepositories().stream()
+              .map(RemoteRepository::getUrl)
+              .collect(toImmutableList());
+      DependencyGraphBuilder dependencyGraphBuilder = new DependencyGraphBuilder(repositoryUrls);
+      classPathBuilder = new ClassPathBuilder(dependencyGraphBuilder);
```

Comment:
We can configure DependencyGraphBuilder to use repositories. (I was forgetting about this feature when I passed MavenProject)